### ISSUE TITLE
draw_figure doesn't take an element

### ISSUE THREAD

**Issue by nikgraf** [jakevdp/mpld3#286]
First of all thanks for this great piece of software @jakevdp 

We are currently introducing mpdl3 to https://github.com/optimamodel/Optima. Here we use a lot of of rendering in the front-end using Angular. One thing we noticed is hat draw_figure doesn't accept elements directly, but rather wants the id. To do this we had to establish a workaround, but adding an unique id to every div where we want to draw a chart.

Our workaround:

    var newId = 'mpld3-chart' + _.uniqueId();
    $(element).attr('id', newId);

What do you think about allowing to provide an ID or the element itself. In this case it wouldn't break the API, but make it easier to work with.

I would be willing to do it myself & submit a PR in case you think it's a good idea.

**Comment by aflaxman** [comment]
I think that this would be a fine addition, so if you want to use it and you can make a PR for it, let's do it.  Are you thinking of a change to [`mpld3.draw_figure`](https://github.com/jakevdp/mpld3/blob/d54a993a4ed4ed1c14ea01bcbda9c989fdca31be/src/utils/util.js#L9)?  

This might be a good opportunity to [revisit how we do functional testing](https://github.com/jakevdp/mpld3/issues/137#issuecomment-40748285) of the javascript part of `mpld3`, if you have any ideas about that...

**Comment by nikgraf** [comment]
@aflaxman Yep, updating `mpld3.draw_figure` would be the plan, but I noticed there is more work to do as the id is passed to other functions as well.
